fec.config.js

```diff
@@ -0,0 +1,28 @@
+const { resolve } = require('path');
+
+module.exports = {
+    appUrl: '/insights/patch',
+    debug: true,
+    useProxy: process.env.PROXY === 'true',
+    proxyVerbose: true,
+    interceptChromeConfig: false,
```

Comment:
This can be omitted as `interceptChromeConfig` is checked to be explicitly set to `true`.

Reply:
@bastilian I have made the necessary changes, PTAL